src/PuliPluginImpl.php

```diff
@@ -200,6 +251,13 @@ public function postInstall()
 
     private function initialize()
     {
+        if (!file_exists($this->autoloadFile)) {
+            $filesystem = new Filesystem();
+            // Avoid problems if using the runner before autoload.php has been
+            // generated
```

Comment:
It doesn't happen directly in this library. Include your PR branch in another project, make sure the `vendor` folder doesn't exist and do a `composer install`.

I get "Warning: Version check failed: PHP Warning:  require_once(/path/to/project/vendor/autoload.php): failed to open stream: No such file or directory in phar:///usr/local/bin/puli/vendor/puli/manager/src/Api/Puli.php on line 313
PHP Fatal error:  require_once(): Failed opening required '/path/to/project/vendor/autoload.php' (include_path='.:') in phar:///usr/local/bin/puli/vendor/puli/manager/src/Api/Puli.php on line 313".